catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -48,46 +43,95 @@ public class MysqlTableOperations extends JdbcTableOperations {
-              .withType(typeConverter.toGravitinoType(columnDefinition.getColDataType()))
-              .withNullable(nullable)
-              .withComment(comment)
-              // TODO: uncomment this once we support column default values.
-              // .withDefaultValue("NULL".equals(defaultValue) ? null : defaultValue)
-              .withProperties(properties)
-              .build());
+    try (Connection connection = getConnection(databaseName)) {
+      // 1.Get table information
+      DatabaseMetaData metaData = connection.getMetaData();
+      ResultSet table = metaData.getTables(databaseName, null, tableName, null);
+      if (!table.next() || !tableName.equals(table.getString("TABLE_NAME"))) {
+        throw new NoSuchTableException(
+            String.format("Table %s does not exist in %s.", tableName, databaseName));
+      }
+      String comment = table.getString("REMARKS");
+
+      // 2.Get index information
+      Map<String, Set<String>> indexGroupByName = new HashMap<>();
+      ResultSet indexInfo = metaData.getIndexInfo(databaseName, null, tableName, false, false);
+      while (indexInfo.next()) {
+        if (!indexInfo.getBoolean("NON_UNIQUE")) {
+          indexGroupByName.compute(
+              indexInfo.getString("COLUMN_NAME"),
+              (k, v) -> {
+                if (v == null) {
+                  v = new HashSet<>();
+                }
+                v.add(UNIQUE_KEY);
+                return v;
+              });
+        }
+      }
+
+      // 3.Get primary key information
+      ResultSet primaryKeys = metaData.getPrimaryKeys(databaseName, null, tableName);
+      while (primaryKeys.next()) {
+        String columnName = primaryKeys.getString("COLUMN_NAME");
+        indexGroupByName.compute(
+            columnName,
+            (k, v) -> {
+              if (v == null) {
+                v = new HashSet<>();
+              }
+              v.add(PRIMARY_KEY);
+              // Remove unique key if exists，because primary key is unique key
+              v.remove(UNIQUE_KEY);
+              return v;
+            });
+      }
+
+      // 4.Get column information
+      ResultSet columns = metaData.getColumns(databaseName, null, tableName, null);
+      while (columns.next()) {
+        MysqlTypeConverter.MysqlTypeBean typeBean =
+            new MysqlTypeConverter.MysqlTypeBean(columns.getString("TYPE_NAME"));
+        typeBean.setColumnSize(columns.getString("COLUMN_SIZE"));
+        typeBean.setDecimalDigits(columns.getString("DECIMAL_DIGITS"));
+
+        String columnName = columns.getString("COLUMN_NAME");
+        String colComment = columns.getString("REMARKS");
+        jdbcColumns.add(
+            new JdbcColumn.Builder()
+                .withName(columnName)
+                .withType(typeConverter.toGravitinoType(typeBean))
+                .withNullable(columns.getBoolean("NULLABLE"))
+                .withComment(StringUtils.isEmpty(colComment) ? null : colComment)
```

Comment:
comment is null and empty are not same, we should replace `StringUtils.isEmpty(colComment) ? null : colComment` with colComment